RunGoogleClassroom.py

```diff
@@ -140,20 +134,19 @@ def Copy_Student_Java_Files(tempDir, service, drive, assignmentSelected, excelWr
 
 def main():
     filter1 = Filter.Filter()
-    (parsed, assignment, xls, addOutput, addFile, goldenSource) = Parse_Args(filter1)
-    (service, drive) = Open_Classroom()        
-    if (parsed):
-        assignmentSelected = Verify_Assignment(service, assignment)
-        if (assignmentSelected != None):  
-            tempPath = RunJavaUtils.Create_Temp_Dir()
-            goldenLines = []
-            if (goldenSource != None):
-                (success, author, package, className, goldenLines) = RunJavaUtils.Copy_And_Run_Java_File(tempPath, goldenSource, None)
-                if (success == False):
-                    print("Build failure for golden source")
-                    return            
-            writer = ExcelWriter.ExcelWriter(xls)                  
-            Copy_Student_Java_Files(tempPath, service, drive, assignmentSelected, writer, addOutput, addFile, goldenLines, filter1)
+    args = Parse_Args(filter1)
+    (service, drive) = Open_Classroom(args)
+    assignmentSelected = Verify_Assignment(service, args.assignment)
+    if assignmentSelected:
```

Comment:
One thing I love about Python is how you can make it read almost like English.  "if assignmentSelected" is I think easier to read than "if (assignmentSelected != None)".

Reply:
In fact `!= None` is a bit of an anti-pattern in Python: 
https://www.python.org/dev/peps/pep-0008/#programming-recommendations.